apps/site/assets/css/_header.scss

```diff
@@ -294,8 +297,89 @@ nav.m-menu {
     display: none;
   }
 }
+
+@mixin dark-mode-links {
```

Comment:
Hrmmm... I got an error, and I think it's because I nested an @extend inside an @include:  "You may not @extend selectors across media queries."  Is there any way to simplify the nested situation?  Or else I think I'm stuck with the mixin...

Reply:
oh! wow I had no idea those worked that way. May as well stick with the mixin! :)